lms/djangoapps/course_api/blocks/tests/test_api.py

```diff
@@ -77,3 +77,32 @@ def test_sub_structure(self):
                 self.assertIn(unicode(block.location), blocks['blocks'])
             else:
                 self.assertNotIn(unicode(block.location), blocks['blocks'])
+
+    def test_visible_to_staff_only(self):
```

Comment:
You may not want to alter the `ToyCourseFactory` code directly since its other clients will expect a certain course structure.  Rather, consider doing a similar thing as is one in the `setUp` method of this code.  Namely, choosing an existing component in the course that you locally update as `visible_to_staff_only`.